flux-plugin/src/flux-runtime/runtime.tsx

```diff
@@ -0,0 +1,112 @@
+/*
+Get all pods and then from them get the controllers for flux
+
+1. helm-controller
+2. kustomize-controller
+3. notification-controller
+4. source-controller
+5. image-reflector-controller
+6. image-automation-controller
+
+*/
+import { K8s } from '@kinvolk/headlamp-plugin/lib';
+import { Link, SectionBox, Table } from '@kinvolk/headlamp-plugin/lib/components/common';
+import { Link as MuiLink } from '@mui/material';
+
+export default function FluxRunTime() {
+  const [pods] = K8s.ResourceClasses.Pod.useList();
```

Comment:
Does this loads all the pods in the cluster? If so this is extremely costly and unnecessary.

I propose we get change this list to show Kubernetes Deployments instead of pods. You can directly filter the Deployments by showing only those with the label `app.kubernetes.io/part-of: flux`.